### PR TITLE
Blog app - setup and controllers

### PR DESCRIPTION
**Pull request by HENRYKC24**
## Following are done in this pull request:
- Created a new app. Gave it a meaningful name.
- Initialized my project with Git.
- Made sure that my project has Postgres database set up.
- Made sure that my app handles the URLs used in the wireframes below:
image
- For each URL I created:
A route.
An action in the correct controller.
A view file.
View files include only placeholders written in plain HTML
Made sure that I added code to handle only the 4 required URLs.

When I run the `rails routes` command in my console, I can see the following output:
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,51 @@
...
 
-* Database creation
+## Getting Started
+Follow the steps bellow to get a local copy up and running.
 
-* Database initialization
+Run `git clone https://github.com/HENRYKC24/Hello-Rails.git` from your terminal to clone the repo from GitHub.
 
-* How to run the test suite
+Run `yarn install` from the command line to install dependencies.
```
Comment (Dipeshtwis) [inline README.md:20]: - [x] I think you have missed some of the steps while writing the steps to run the project locally.

Like after cloning the repo, make sure you go to right folder, then add `username` and `password` according to your local postgresql setup, if you don't have please install it. 

```ruby
$ git clone https://github.com/HENRYKC24/Hello-Rails.git

$ cd Hello-Rails
$ bundle install

# Set username and password according to your postgresql in `config/database.yml`

# To create a database
$ rails db:create

# Below command is optional for this repo, but it is mandatory from the milestone where you will add the migration

$ rails db:migrate

# When you use seed data
$ rails db:seed

# then run server, which you have already mentioned 

$ rails s
```

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @HENRYKC24, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (HENRYKC24) [reply README.md:20]: Thank you very much:pray:
Be sure to have me include that right away:running:

**Review by Emmyn5600 — APPROVED** [review]
Hi @HENRYKC24 

Wooow👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

##  Optional suggestion

-  N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by HENRYKC24** [comment]
Thank you.

**Pull request merged**